src/services/fetchers/CryptoComparePriceWSFetcher.ts

```diff
@@ -2,19 +2,20 @@ import {inject, injectable} from 'inversify';
 
 import CryptoCompareWSClient from '../ws/CryptoCompareWSClient.js';
 import {PairWithFreshness} from '../../types/Feed.js';
+import {FeedFetcherInterface} from 'src/types/fetchers.js';
 
 @injectable()
-class CryptoComparePriceWSFetcher {
+class CryptoComparePriceWSFetcher implements FeedFetcherInterface {
   @inject(CryptoCompareWSClient) cryptoCompareWSClient!: CryptoCompareWSClient;
 
-  async apply(pair: PairWithFreshness, timestamp: number): Promise<number> {
-    const price = await this.cryptoCompareWSClient.getLatestPrice(pair, timestamp);
+  async apply(params: PairWithFreshness, base: string, quote: string, timestamp: number): Promise<number> {
```

Comment:
```suggestion
  async apply(params: PairWithFreshness, options: PriceFetherOptions): Promise<number> {
```

Reply:
addressed, I just called it `FeedFetcherOptions` for consistency but I can change it if you prefer.